msmbuilder/msm/core.py

```diff
@@ -473,7 +478,13 @@ def _transition_counts(sequences, lag_time=1, sliding_window=True):
         shape=(n_states, n_states))
     counts = counts + np.asarray(C.todense())
 
-    return counts / float(lag_time), mapping
+    # The following condition isn't strictly necessary: if sliding window
```

Comment:
I find it hard to understand what this comment is saying.  Is the code unnecessary?  If so, can we delete it?